MyPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+func punkt_1(a: Double, b: Double, c: Double) -> [Double]? {//решение квадратного уравнения
+
+    let dscr = b * b - 4 * a * c
+
+    switch dscr{
+    case 0:
+        let x_1 = -b / 2 * a
+        return [x_1]
+    case 0...:
+        let x_1 = -b - sqrt(dscr) / 2 * a
+        let x_2 = -b + sqrt(dscr) / 2 * a
+        return [x_1, x_2]
```

Comment:
И тут аналогично убрать 2 строчки выше и:
```suggestion
        return [ -b - sqrt(dscr) / 2 * a, -b + sqrt(dscr) / 2 * a]
```